haas/cli.py

```diff
@@ -444,6 +474,27 @@ def stop_console(node):
     do_delete(url)
 
 @cmd
+def create_admin_user(username, password):
```

Comment:
Aha. this seems to be the initialization process referred to in INSTALL.rst. Not sure how it gets invoked.